content/en/docs/setup/install/kubernetes/index.md

```diff
@@ -35,15 +26,13 @@ requirements.
 
 ## Installation steps
 
-1. Install all the Istio
-    [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-    (CRDs) using `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kubernetes API-server:
+1. Install Istio using the operator with the default profile:
 
     {{< text bash >}}
-    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+    $ kubectl apply -f https://preliminary.istio.io/operator.yaml
```

Comment:
Should be just istio.io (even though the link will not be functional until we publish the 1.4 doc set)

Reply:
@geeknoid I wish people to be able to test from the preliminary docs.  If the preliminary docs work well, this can easily be modified prior to the cut of istio 1.4.

Reply:
Is there no way to reference "the current site" as a variable? Which could be istio.io, preliminary, netlify deploy preview, etc?

Reply:
For normal links, we just use relative paths (e.g. /docs/foo) and those resolve correctly. But this is a preformatted block and that kind of resolution is not available. There's no way currently to specify this kind of auto-correcting reference.

Reply:
Why are the operator profile yaml files published on istio.io in the first place? They're not documentation. Shouldn't they be in the release?

Reply:
@frankbu See: https://github.com/istio/istio.io/pull/5142#issuecomment-541419564

the requirements are as follows:
- the yaml files must be downloadable with `kubectl apply -f https://istio.io/operator.yaml`
-  happy to use a redirect such as https://operator.istio.io
-  Agree it would be preferrable to place in a GCS bucket, however, I don't have said permissions to do that.  If someone would be kind enough to point me at an example, I'll make that happen.
-  Even in a GCS bucket (which could be automated per release) - the redirect would need to work and kubectl would need to follow the redirect.

We don't know if we can meet the above requirements with GCS.  It is worth investigation.

Cheers
-steve